### DISCUSSION TITLE
Chirpity v0.11.0 released

### DISCUSSION THREAD

**Discussion by Mattk70** [Mattk70/Chirpity-Electron#19] [Announcements]
I am delighed to say that I have just released a version of Chirpity which integrates the latest BirdNET model (v2.4). I owe a huge debt to @Kahst for his support, without which this integration would not have been possible.

I hope you find this release useful, and truly hope it helps unlock the field of nocmig and bioacoutsic monitoring to a wider audience!

**Comment by kahst** [comment]
Awesome! We'd love to feature Chirpity in our "showroom" on our BirdNET-Analyzer GitHub page. Do you have a preferred icon we should use?

**Reply by Mattk70** [reply]
@kahst, that's amazing, what an honour, thank you! I think this would be the best logo to use:

![chirpity_logo2](https://github.com/Mattk70/Chirpity-Electron/assets/61826357/a1e0f524-80c7-4145-9a30-cd007969d8d5)

**Reply by kahst** [reply]
Great, keep up the good work! I'm a strong believer in web technologies, yet, not skilled enough to build stuff in JavaScript :) 
Let me know if you want to change the showroom entry.

https://github.com/kahst/BirdNET-Analyzer/blob/main/README.adoc#4-showroom

**Reply by Mattk70** [reply]
That's great Stefan, thank you! I might have to have a think about the description, because, now it has BirdNET integration


![one-does-not-6f6da16736](https://github.com/Mattk70/Chirpity-Electron/assets/61826357/e90c9652-c0c5-4034-8138-28f41baa571e)

**Reply by kahst** [reply]
Sure :) Just let us know. Or create a PR.

**Comment by DD4WH** [comment]
@Mattk70 
Downloaded and installed version v0.11.0 and tried it this afternoon. 
Fantastic GUI, very easy to use and -as far as I can tell from the few recordings I tried- with consistent results compared to BirdNET-Analyzer GUI !

Thank you very much for this magnificient piece of software! Very good, how you integrated the location filter (local birds) and the time filter (NocMig). This makes it easy to use and understand ! The nice design look of the program and the very fast response (and the possibility to look at the preliminary results even during an ongoing analysis) makes it fun and entertaining to use. Thanks also for the bugfix of the remaining-files-to-process display!  

There are some small wishes for a future version from my side, which I will communicate with a github issue !

Keep up the good work and thanks again for Chirpity!

Frank DD4WH

**Comment by Vollpflock** [comment]
Totally agree with DD4WH. I was able to try it with a friends laptop today and I am blown away!
